From 03eaf3b9bdf7ce04f1285d08ee74af63ca8fd46b Mon Sep 17 00:00:00 2001
From: Stuart Morgan <smorgan@mythtv.org>
Date: Wed, 14 Jan 2015 23:10:58 +0000
Subject: [PATCH] Icon Downloader: Fix improper escaping of search string that
 broke proper matching and sorting with some searches

(cherry picked from commit 93f4b96128c454aac728301282259df6734eb771)
---
 mythtv/programs/mythtv-setup/importicons.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/mythtv/programs/mythtv-setup/importicons.cpp b/mythtv/programs/mythtv-setup/importicons.cpp
index 6cceb04..ec8e506 100644
--- a/mythtv/programs/mythtv-setup/importicons.cpp
+++ b/mythtv/programs/mythtv-setup/importicons.cpp
@@ -165,8 +165,8 @@ void ImportIconsWizard::enableControls(dialogState state, bool selectEnabled)
 void ImportIconsWizard::manualSearch()
 {
     QString str = m_manualEdit->GetText();
-    if (!search(escape_csv(str)))
-        m_statusText->SetText(tr("No matches found for %1") .arg(str));
+    if (!search(str))
+        m_statusText->SetText(tr("No matches found for \"%1\"").arg(str));
     else
         m_statusText->Reset();
 }
@@ -476,7 +476,7 @@ bool ImportIconsWizard::doLoad()
     m_nameText->SetText(tr("Choose icon for channel %1")
                         .arg((*m_missingIter).strName));
     m_manualEdit->SetText((*m_missingIter).strName);
-    if (!search((*m_missingIter).strNameCSV))
+    if (!search((*m_missingIter).strName))
         m_statusText->SetText(tr("No matches found for %1")
                               .arg((*m_missingIter).strName));
     else
